.changeset/seven-bobcats-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"comet-admin-v5": major
```

Comment:
why is this a major change? shouldnt this be a minor?

Reply:
This was the output from running `npx changeset` for the given Chart. Should I change it to minor for all PRs ?

Reply:
Version was changed to minor